src/DN2.jl

```diff
@@ -0,0 +1,107 @@
+module DN2
+
+    using LinearAlgebra
+
+    """Izračun binomskega koeficienta za n, k."""
+    function binom(n, k)
+        factorial(n) ÷ (factorial(k) * factorial(n - k))
+    end
+
+    """Polinomsko množenje polinomov p in q, predstavljenih kot polje koeficientov."""
```

Comment:
Nima smisla, da računate s koeficienti v standardni bazi. Če že uporabljate formulo z Bernsteinovo bazo, lahko tudi računate direktno z Bernsteinovimi baznimi polinomi. Formula je preprosta, saj zgolj množite potence `t` in `1-t`.